webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criterion-rating-edit.js

```diff
@@ -153,6 +154,13 @@ export class SakaiRubricCriterionRatingEdit extends RubricsElement {
       this.editRating(e);
     }
   }
+
+  removeButtonTitle() {
```

Comment:
I know some people find this pedantic and weird, but, please indulge and old man and leave a blank line after the function name.